.github/workflows/package-server.yml

```diff
@@ -141,9 +144,18 @@ jobs:
       - name: Generate requirements & constraints infos
         run: python server/packaging/wheel/wheel_it.py ./server --output dist --skip-wheel
 
+      # Install syft
+      - uses: taiki-e/install-action@486baeb8af63bc3d9ec3ec66db5af6ba0ca78774 # pin v2.11.6
+        with:
+          tool: syft@0.84.0
+
+      - name: Generate SBOM
+        run: syft packages --config=.syft.yaml --output=spdx-json=dist/Parsec-Wheel-${{ matrix.platform }}.spdx.json .
```

Comment:
It is uploaded, the generated SBOM is put in the `dist/` folder that is then uploaded as a whole

Reply:
> why it is generated in dist/ instead of ${{ runner.temp }} like the other SBOM below?

I depend on how we build the artifact, some here build in the temp folder some other are not (like here). the goal is to execute `syft` where the generated artifacts lays